cogboard-webapp/src/components/widgets/types/PullRequestReminderWidget/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import InfoOutlinedIcon from '@material-ui/icons/InfoOutlined';
+
+import { formatPullRequestTitle, preparePullRequestArray } from './helpers.js';
+
+import { StyledNoItemsInfo } from '../../../Widget/styled';
+import { PullRequestContainer, PullRequestLink } from './styled';
+
+const PullRequestReminderWidget = ({ repositoryHub, pullRequests }) => {
+  pullRequests = preparePullRequestArray(repositoryHub, pullRequests);
+
+  if (!pullRequests) {
+    return (
+      <StyledNoItemsInfo>
+        <InfoOutlinedIcon fontSize="large" />
+        <p>API rate limit exceeded!</p>
+      </StyledNoItemsInfo>
+    );
+  }
+
+  return (
+    <>
+      {pullRequests.length > 0 ? (
+        pullRequests.map(({ id, title, url }, index) => (
+          <PullRequestLink key={id} href={url} target="_blank">
+            <PullRequestContainer>
+              {`${index + 1}. ${formatPullRequestTitle(title)}`}
+            </PullRequestContainer>
+          </PullRequestLink>
```

Comment:
Please reuse the existing look & feel from other widgets - look at Random picker, Link List, ToDo list

Reply:
Fixed in commit fe5980b